101-create-applicationgateway-publicip/azuredeploy.json

```diff
@@ -0,0 +1,199 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location": {
+            "type": "string",
+            "metadata": {
+                "description": "Location to deploy to"
+            }
+        },
+        "applicationGatewayName": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of Application Gateway"
+            }
+        },
+        "publicIPAddressName": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of Public IP address resource"
+            }
+        },
+        "virtualNetworkName": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of Virtual Network"
+            }
+        },
+        "subnetName": {
```

Comment:
This can become a hardcoded variable to reduce the number of parameters needed to deploy this template